### PR TITLE
Fix hang bug of TCPStore

### PR DESCRIPTION
**Pull request by gongweibao**
### PR types
 Bug fixes 

### PR changes
Others

### Describe
TCPStore hangs occasionally since it doesn't have pipe control strategy.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_tcp_store.py [hunk 1]
```diff
@@ -22,8 +22,9 @@
 class TestTCPStore(unittest.TestCase):
 
     def test_tcp_store(self):
-        store = paddle.fluid.core.TCPStore("127.0.0.1", 6170, True, 1,
-                                           datetime.timedelta(0))
+        dist_port = os.getenv("PADDLE_DIST_UT_PORT", 6170)
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_tcp_store.py]: import os maybe?

**Review by JiabinYang — COMMENTED** [review]
one comment

File: python/paddle/fluid/tests/unittests/test_tcp_store.py (hunk 1 above)
Comment (gongweibao) [reply python/paddle/fluid/tests/unittests/test_tcp_store.py]: Done

**Review by XieYunshen — APPROVED** [review]
LGTM 单测名称修改

**Review by fuyinno4 — APPROVED** [review]

**Review by chalsliu — APPROVED** [review]

**Review by chenwhql — APPROVED** [review]

**Review by fuyinno4 — APPROVED** [review]

**Pull request merged**